### PR TITLE
Add aliases for Visual C built in types extension

### PR DESCRIPTION
**Pull request by 12345swordy**
https://docs.microsoft.com/en-us/cpp/cpp/int8-int16-int32-int64?view=msvc-170

### CODE REVIEW

**Comment by 12345swordy** [comment]
cc @ibuclaw @kinke will this negatively effect ldc/gdc here?

File: src/__builtins.di [hunk 1]
```diff
@@ -38,3 +38,12 @@ alias __builtin_va_copy = core.stdc.stdarg.va_copy;
...
 alias va_arg = core.stdc.stdarg.va_arg;
+
+version (Windows)
+{
+    //https://docs.microsoft.com/en-us/cpp/cpp/int8-int16-int32-int64?view=msvc-170
+    alias __int8 = byte;
+    alias __int16 = short;
+    alias __int32 = int;
+    alias __int64 = long;
+}
```
Comment (thewilsonator) [inline src/__builtins.di:49]: newline before EOF

**Comment by kinke** [comment]
This shouldn't affect any backend. - version `CRuntime_Microsoft` might be more appropriate, as mingw-gcc probably doesn't have these builtins.

File: src/__builtins.di [hunk 2]
```diff
@@ -38,3 +38,12 @@ alias __builtin_va_copy = core.stdc.stdarg.va_copy;
 /* dmd's ImportC rewrites __builtin_va_arg into an instantiation of va_arg
  */
 alias va_arg = core.stdc.stdarg.va_arg;
+
+version (Windows)
```
Comment (PetarKirov) [inline src/__builtins.di:42]: Perhaps this should be:
```suggestion
version (CRuntime_Microsoft)
```

**Review by thewilsonator — APPROVED** [review]

**Pull request merged**